authz/manifests.go

```diff
@@ -39,4 +39,70 @@ subjects:
 - kind: ServiceAccount
   namespace: kube-system
   name: rke-job-deployer`
+
+	// DefaultPodSecurityPolicy is very permissive, look into reducing this.
```

Comment:
does this comment still apply? given that the policy is limited to kube-system namespace, and we are not gonna reduce the permissions for those